controllers/usersController.js

```diff
@@ -3,14 +3,14 @@ const { getAllUsers, getUserById } = require('../queries/usersQueries');
 
 const usersController = express.Router();
 
-usersController.get('/', (req, res) => {
-  users = getAllUsers() || [];
+usersController.get('/', async (req, res) => {
+  users = await getAllUsers() || [];
```

Comment:
This is my bad, but the original code should have `const` or `let` before `users`.